python/hail/expr/functions.py

```diff
@@ -5,7 +5,7 @@
 from hail.genetics import Variant, Locus, Call, GenomeReference
 
 expr_int32 = oneof(Int32Expression, int)
-expr_numeric = oneof(Float32Expression, Float64Expression, Int64Expression, float, expr_int32)
+expr_numeric = oneof(Float32Expression, Float64Expression, Int64Expression, float, int, expr_int32)
```

Comment:
Why are int and Int32Expression handled differently here?

Reply:
I didn't realize expr_int32 was here. Will revert!